tests/testthat/test-config-formulas.R

```diff
@@ -0,0 +1,67 @@
+context("config")
+
+input <- list(population = 1000,
```

Comment:
can you move this top-level code into a function, as it will otherwise pollute the namespace for all tests that follow it (scope is not limited to this file, unfortunately)